rebasehelper/specfile.py

```diff
@@ -427,10 +414,9 @@ def write_updated_patches(self, patches, disable_inapplicable):
                     self.spec_content[index] = ' '.join(fields) + '\n'
                     modified_patches.append(patch_num)
 
-        self._comment_out_patches(removed_patches)
-        note = 'Disabled because of conflicts, please verify'
-        self._comment_out_patches(inapplicable_patches, note=note)
-        self._correct_rebased_patches(modified_patches)
+        note = 'Following patch contains conflicts'
+        self._process_patches(inapplicable_patches, removed_patches, note, disable_inapplicable)
```

Comment:
I think you can move the `note` constant into the `_process_patches` method, it's pointless to keep it here now.